envi/archs/arm/disasm.py

```diff
@@ -2668,6 +2796,14 @@ def setArchMask(self, key = 'ARMv7R'):
         self._archVersionMask = 0
         if key in ARCH_REVS:
             self._archVersionMask = ARCH_REVS[key]
```

Comment:
make this function:

self._archVersionMask = ARCH_REVS.get(key, 0)

and get rid of all the if/elif/else.  i realize this code is on hold, but let's fix this since it's in the PR.
then move the REV_THUMB16/2 into the ARCH_REVS as well.